spk/lidarr/src/service-setup.sh

```diff
@@ -17,33 +19,45 @@ SVC_WAIT_TIMEOUT=90
 service_postinst ()
 {
     # Move config.xml to .config
-    mkdir -p ${CONFIG_DIR}/Lidarr
-    mv ${SYNOPKG_PKGDEST}/app/config.xml ${CONFIG_DIR}/Lidarr/config.xml
-    set_unix_permissions "${CONFIG_DIR}"
+    mkdir -p ${LIDARR_CONFIG_DIR}
+    mv ${SYNOPKG_PKGDEST}/app/config.xml ${LIDARR_CONFIG_DIR}/config.xml
+
+    echo "Set update required"
+    # Make Lidarr do an update check on start to avoid possible Lidarr
+    # downgrade when synocommunity package is updated
+    touch ${LIDARR_CONFIG_DIR}/update_required
```

Comment:
Also require users to have enabled auto updates (disabled by default)

but presumably most users would have it enabled or otherwise see the healthcheck saying to update

Reply:
How do we force enable Auto update for all the *arr programs? Is there a way?
Or at least for new installations.

Reply:
@bakerboy448 

the config.xml installed in the lidarr package has:
```xml
<UpdateMechanism>BuiltIn</UpdateMechanism>
```

Is this enough to  enable auto updates?

Reply:
Or does one need to configure an RSS indexer therefore?
```
ERROR: No indexers available with RSS sync enabled, Lidarr will not grab new releases automatically
```

Reply:
just found id (it was hidden in the advanced general settings)

If automatic updates are enabled, the config.xml contains:
```xml
<UpdateAutomatically>True</UpdateAutomatically>
```